src/Tools/DynamoInstallDetective/ProductLookUp.cs

```diff
@@ -120,38 +232,13 @@ public interface IProductCollection
     /// </summary>
     public class InstalledProductLookUp : IProductLookUp
     {
-        const string REG_KEY64 = @"SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\";
-        const string REG_KEY32 = @"SOFTWARE\Wow6432Node\Microsoft\Windows\CurrentVersion\Uninstall\";
-        
         /// <summary>
         /// Product name for lookup
         /// </summary>
         public string ProductLookUpName { get; private set; }
 
         private readonly Func<string, string> fileLocator;
 
-        static RegistryKey OpenKey(string key)
```

Comment:
default access modifiers should be private...so we should be good with this change